<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GitHub Metrics Report</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="reportTabs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>741</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="summaryStatisticsTab">
     <attribute name="title">
      <string>Summary Statistics</string>
     </attribute>
     <widget class="QTableWidget" name="summaryTable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>541</width>
        <height>461</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <row>
       <property name="text">
        <string>Days</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>A</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>B</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>C</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>D</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>E</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>F</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>G</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>H</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>I</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>J</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>K</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>M</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Median</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>25th</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mean</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>75th</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stdev</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="metricA">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricDay">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricC">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricD">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricF">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricG">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricI">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricJ">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricK">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricL">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>410</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="metricM">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>440</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>metric_detail_icon.png</normaloff>metric_detail_icon.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Basic summary statistics for each metric. Click the detail button beside each metric for more detail.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lowestRankTab">
     <attribute name="title">
      <string>Lowest Ranked</string>
     </attribute>
     <widget class="QTableWidget" name="lowestRankTable">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>541</width>
        <height>451</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <row>
       <property name="text">
        <string>Days</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>A</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>B</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>C</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>D</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>E</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>F</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>G</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>H</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>I</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>J</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>K</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>M</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1st</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2nd</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3rd</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4th</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5th</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>user1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>user5</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>user10</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>user8</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>user12</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>user2</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>user6</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>user1</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>user9</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>user19</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>The five lowest ranked users for each metric</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="fileNameText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Loaded: F2020.csv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="loadCSVAction"/>
    <addaction name="closeAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="closeAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="loadCSVAction">
   <property name="text">
    <string>Load CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
